core-common/src/main/java/org/mapstruct/CollectionMappingStrategy.java

```diff
@@ -51,5 +52,12 @@
      * Identical to {@link #SETTER_PREFERRED}, only that adder methods will be preferred over setter methods, if both
      * are present for a given collection-typed property.
      */
-    ADDER_PREFERRED;
+    ADDER_PREFERRED,
+
+    /**
+     * Identical to {@link #SETTER_PREFERRED}, however a target collection will not cleared be accessed via addAll.
+     * {@code orderDto.getOrderLines().clear(); orderDto.getOrderLines().addAll(order.getOrderLines)}. Instead the
```

Comment:
Maybe we should write the code with:

```
<pre>
orderDto.getOrderLines().clear();
orderDto.getOrderLines().addAll(order.getOrderLines);
</pre>
```

This way it will be on 2 lines when viewing the Javadoc